geo/src/algorithm/bounding_rect.rs

```diff
@@ -141,13 +156,72 @@ where
     }
 }
 
+impl<T> BoundingRect<T> for Geometry<T>
+where
+    T: CoordinateType,
+{
+    type Output = Option<Rect<T>>;
+
+    fn bounding_rect(&self) -> Self::Output {
```

Comment:
Take it or leave it, or possibly for another PR, but can this can return a Rect instead of an Option<Rect>?

Or is there still something returning a `None` bounding rect?

Reply:
completely agree with you that it should return a `Rect`, but there are existing implementations (like `impl BoundingRect for LineString`) that will return `None` if the `LineString` has zero coordinates. we should change this in the future though!